### PR TITLE
Review feature

### PR DESCRIPTION
**Pull request by janegreene**
Completed user stories 2, 3, and 5
Site up and running on Heroku (user story 1)
skipped 4, waiting on lesson

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -17,3 +17,4 @@
 /yarn-error.log
 
 .byebug_history
+coverage/
```
Comment (janegreene) [inline .gitignore:20]: Added this to ignore...rolling with the modified of the reports we already committed.

File: app/models/shelter.rb [hunk 2]
```diff
@@ -1,4 +1,5 @@
 class Shelter < ApplicationRecord
   validates_presence_of :name
   has_many :pets, dependent: :delete_all
+  has_many :reviews, dependent: :delete_all
```
Comment (janegreene) [inline app/models/shelter.rb:4]: This relationship is critical to functionality.

File: app/controllers/reviews_controller.rb [hunk 3]
```diff
@@ -0,0 +1,28 @@
+class ReviewsController < ApplicationController
+  def new
+    @shelter = Shelter.find(params[:id])
+  end
+
+  def create
+    shelter = Shelter.find(params[:id])
+    review = shelter.reviews.create!(review_params)
+
+    redirect_to "/shelters/#{shelter.id}"
```
Comment (whitneykidd) [inline app/controllers/reviews_controller.rb:10]: So glad we finally got this to work! 🎉

File: app/views/reviews/edit.html.erb [hunk 4]
```diff
@@ -0,0 +1,18 @@
+<div class="form_container">
+<h1>Edit Review</h1>
+<%= form_tag "/shelters/#{@review.shelter_id}/#{@review.id}", method: :patch do %>
+
+  <%= label_tag :title, "Title" %><br>
```
Comment (janegreene) [inline app/views/reviews/edit.html.erb:5]: This style will show previous entries.

File: app/models/shelter.rb (hunk 2 above)
Comment (whitneykidd) [inline app/models/shelter.rb:4]: This delete_all dependency will come in handy for reviews#delete

**Pull request merged**